tests/checkbox.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from "@playwright/test";
+
+test("The joke is complete, type of boolean equal to true with a checkbox", async ({
```

Comment:
great beginning test, I think this indicates more to if it's falsy rather than truthy on checkbox ✅